pkg/rbd/rbd_util.go

```diff
@@ -227,24 +240,79 @@ func execCommand(command string, args []string) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+func getMonsAndFsID(options map[string]string) (monitors, fsID, monInSecret string, noerr error) {
+	var err error
+	var ok bool
+
+	monitors, ok = options["monitors"]
+	if !ok {
+		// if mons are not set in options, check if they are set in secret
+		if monInSecret, ok = options["monValueFromSecret"]; !ok {
+			// if mons are not in secret, check if we have a cluster-fsid
+			if fsID, ok = options["clusterID"]; !ok {
+				return "", "", "", fmt.Errorf("either monitors or monValueFromSecret or clusterID must be set")
```

Comment:
i think we should not mix named return  and agrument return
i think we can just do as below
```
noerr=errors.New(("either monitors or monValueFromSecret or clusterID must be set")
return
```

Reply:
I did not find literature to contradict or support your statement on mixed named and naked returns, but I did address the comment, as it was looking odd :)